ci-operator/step-registry/ipi/conf/aws/user-provisioned-dns/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
Let us add Rafael because he is highly active in AWS platform as a whole and me because I am deeply invested in this feature.